src/command.ts

```diff
@@ -0,0 +1,47 @@
+import { isFunction, isString } from 'util'
+
+const commandParse: RegExp = /^(?:(\w+)\.)?(?:(\w+):)?(\w+)$/
+
+export default class Command {
```

Comment:
Command class is expression `_ud()` arguments.

Reply:
why use class ?
this class instance variables read only state?